### ISSUE TITLE
import of mirage upload handler causes fastboot error

### ISSUE THREAD

**Issue by gibso** [adopted-ember-addons/ember-file-upload#377]
I am using this addon in a large ember-application with fastboot and mirage. So far, it works like a charm. But as soon as i want to import the upload helper in my mirage config ...

```javascript
// mirage/config.js

import { upload } from 'ember-file-upload/mirage';
``` 
... Fastboot throws an error when i call any route in my browser. When i turn off fastboot everything works fine. Am i missing something?

Thank you very much in advance!

```
ReferenceError: FormData is not defined
    at Module.callback (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/addon-tree-output/ember-file-upload/mirage/shim.js:11:1)
    at Module.exports (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:106:1)
    at Module._reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:104:1)
    at Module._reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:104:1)
    at Module._reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:104:1)
    at requireModule (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:27:1)
    at resolveInitializer (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/addon-tree-output/ember-load-initializers/index.js:34:1)
    at registerInitializers (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/addon-tree-output/ember-load-initializers/index.js:47:1)
    at exports.default (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/addon-tree-output/ember-load-initializers/index.js:28:1)
    at Module.callback (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/eos/app.js:51:1)
    at Module.exports (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:106:1)
    at Module._reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:143:1)
    at Module.reify (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:130:1)
    at Module.exports (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:104:1)
    at Object.requireModule (/tmp/broccoli-1COZmebBW7o5T/out-684-broccoli_merge_trees/assets/vendor/loader/loader.js:27:1)
    at Object.<anonymous> (/var/workdir/node_modules/fastboot/src/ember-app.js:213:18)
    at VMSandbox.run (/var/workdir/node_modules/fastboot/src/vm-sandbox.js:18:15)
    at EmberApp.createEmberApp (/var/workdir/node_modules/fastboot/src/ember-app.js:212:30)
    at EmberApp.retrieveSandboxedApp (/var/workdir/node_modules/fastboot/src/ember-app.js:237:17)
    at new EmberApp (/var/workdir/node_modules/fastboot/src/ember-app.js:61:21)
    at FastBoot._buildEmberApp (/var/workdir/node_modules/fastboot/src/index.js:114:17)
    at new FastBoot (/var/workdir/node_modules/fastboot/src/index.js:52:10)
    at /var/workdir/node_modules/ember-cli-fastboot/index.js:330:29
    at Layer.handle [as handle_request] (/var/workdir/node_modules/ember-cli/node_modules/express/lib/router/layer.js:95:5)
    at trim_prefix (/var/workdir/node_modules/ember-cli/node_modules/express/lib/router/index.js:317:13)
    at /var/workdir/node_modules/ember-cli/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/var/workdir/node_modules/ember-cli/node_modules/express/lib/router/index.js:335:12)
    at next (/var/workdir/node_modules/ember-cli/node_modules/express/lib/router/index.js:275:10)
    at /var/workdir/node_modules/ember-cli/lib/tasks/server/middleware/broccoli-watcher/index.js:54:11
    at watcher.then.errorHandler.buildError (/var/workdir/node_modules/broccoli-middleware/lib/watcher-middleware.js:35:7)
    at tryCatcher (/var/workdir/node_modules/ember-cli/node_modules/rsvp/dist/lib/rsvp/-internal.js:39:19)
    at invokeCallback (/var/workdir/node_modules/ember-cli/node_modules/rsvp/dist/lib/rsvp/-internal.js:211:31)
    at /var/workdir/node_modules/ember-cli/node_modules/rsvp/dist/lib/rsvp/then.js:26:14
    at flush (/var/workdir/node_modules/ember-cli/node_modules/rsvp/dist/lib/rsvp/asap.js:80:5)
    at processTicksAndRejections (internal/proces
              
```

**Comment by gibso** [comment]
Hey @jelhan, thanks for your reply.

Well, i think that must be the cause. The first line of the stacktrace points to this file
```
    at Module.callback (/tmp/.../assets/addon-tree-output/ember-file-upload/mirage/shim.js:11:1)

```
i can just try to delete this file and see if everything works fine.

**Comment by gibso** [comment]
So i tried to delete it and indeed, it fixes the error. I can not really say if this file is completely obsolete then. But i prepared a pull request in case it is.

**Comment by jelhan** [comment]
Just released [v3.0.5](https://github.com/adopted-ember-addons/ember-file-upload/releases/tag/v3.0.5), which contains the fix in #378. Going to close this issue now. Please reopen if you are still facing errors in FastBoot.
